charts/consul/values.yaml

```diff
@@ -3069,3 +3074,86 @@ prometheus:
 # is only useful when running helm template.
 tests:
   enabled: true
+
+telemetryCollector:
+  # Enables the consul-telemetry-collector deployment
+  # @type: boolean
+  enabled: false
+
+  # The name of the Docker image (including any tag) for the containers running
+  # the consul-telemetry-collector
+  # @type: string
+  image: "hashicorp/consul-telemetry-collector:0.0.1-dev"
```

Comment:
Since all of the scripts assume that the they're operating on _this_ git repository it sounds like it'll be simpler to include instructions in the release process to grab the latest docker tag.

Reply:
Sounds good!